content/docs/1.0.0/tutorial/get-to-know-iofog.md

```diff
@@ -4,48 +4,44 @@ In this step of the tutorial we'll better familiarize ourselves with the ioFog e
 
 ## Docker Environment
 
-To verify everything is up and working correctly, run `docker ps`:
-
-```sh
-docker ps
+Verify manually that the ioFog stack containers are correctly started. Note that it may take a minute or two for ioFog to start the tutorial microservices. The output should look like this.
+
+```console
+$ docker ps --filter "name=iofog"
+
+CONTAINER ID IMAGE                      COMMAND                CREATED            STATUS            PORTS                    NAMES
+baefdede1949 iofog/freeboard:latest     "nginx -g 'daemon of…" 14 seconds ago     Up 13 seconds     0.0.0.0:10102->80/tcp    iofog_BrKHZf9PTcT6yjKcrpnRVBcYPFqxqXxb
+ce6d14551a57 iofog/freeboard-api:latest "node /src/index.js"   17 seconds ago     Up 16 seconds     0.0.0.0:10101->80/tcp    iofog_NZp8HZ7xpztPyC4dpRQx4w3Jd8x9jNF3
+43c00c7c9fc0 iofog/sensors:latest       "node /sensors/index"  20 seconds ago     Up 19 seconds                              iofog_PDRRQcbD6DVZJy9QBJQB7JyzH7RgLJCb
+5df0afa1018a demo_iofog-agent           "/usr/bin/supervisord" About a minute ago Up About a minute 0.0.0.0:8081->22/tcp     iofog-agent
+7146cd77d400 demo_iofog-controller      "sh -c 'iofog-contro…" About a minute ago Up About a minute 0.0.0.0:51121->51121/tcp iofog-controller
+9e64f5e1afd4 demo_iofog-connector       "sh -c 'service iofo…" About a minute ago Up About a minute 0.0.0.0:53321->8080/tcp  iofog-connector
 ```
 
-You should see a number of containers running.
-
 We can also see that in the PORTS column some of these containers have published [port mappings](https://docs.docker.com/config/containers/container-networking/):
 
-```sh
-PORTS                      NAMES
-0.0.0.0:51121->51121/tcp   iofog-controller
-                           iofog-agent-1
-                           iofog-agent-2
-0.0.0.0:53321->8080/tcp    iofog-connector
-```
-
-For example, there's a microservice running an instance of [Freeboard](https://github.com/Freeboard/freeboard), an open source visualization dashboard for IoT devices. It's running a web server that serves the dashboard, try it out: http://localhost:10102/?load=dashboard.json
+For example, there's a microservice running an instance of [Freeboard](https://github.com/Freeboard/freeboard), an open source visualization dashboard for IoT devices. It's running a web server that serves the dashboard, try it out at http://localhost:10102/?load=dashboard.json
```

Comment:
It's running a web server that serves the dashboard**.** **Go ahead and** try it out at http://localhost:10102/?load=dashboard.json